src/components/TradeWidget/OrderDetails.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useFormContext } from 'react-hook-form'
+
+import { TokenDetails } from 'types'
+import { FEE_PERCENTAGE } from 'const'
+
+const Wrapper = styled.dl`
```

Comment:
Yeah, that's a thing I'm still conflicted about. As I get familiar with css/styled-components I'm trying different approaches. When I wrote this it did feel weird, but I pushed anyway to get feedbacks like this one :)

I'll refactor in a more compact approach.

Reply:
Done. I kept the `Highlight` element though, as it clears up clutter in the tsx code without all the `<span className="highlight">` repetition.

Reply:
Okay, thanks!
I'm the same. I also try different approaches as I get familiar, and since I'm not 100% sure either about this cases I was opening a small discussion in case someone has an opinion.

cc/ @Velenir @W3stside